pkg/cache/v3/snapshot_test.go

```diff
@@ -112,6 +112,30 @@ func TestScopedRouteListenerWithScopedRouteAndRouteIsConsistent(t *testing.T) {
 	}
 }
 
+func TestScopedRouteListenerWithInlineScopedRouteAndRouteIsConsistent(t *testing.T) {
+	snap, _ := cache.NewSnapshot(version, map[rsrc.Type][]types.Resource{
+		rsrc.ListenerType: {
+			resource.MakeScopedRouteHTTPListenerForRoute(resource.Xds, "listener0", 80, "scopedRoute0", "testRoute0"),
+		},
+		rsrc.RouteType: {
+			resource.MakeRoute("testRoute0", clusterName),
+		},
+	})
+
+	assert.NoError(t, snap.Consistent())
+
+	snap, _ = cache.NewSnapshot(version, map[rsrc.Type][]types.Resource{
```

Comment:
NIT: The title only describes the first test case here for a consistent snapshot, but the second case is not referenced. Could you either update the test title, or break this test case in 2?

Reply:
I split it into 2.